tests/Aspire.Hosting.Tests/ManifestGenerationTests.cs

```diff
@@ -97,15 +97,15 @@ public async Task WithContainerRegistryUpdatesContainerImageAnnotationsDuringPub
             ContainerRegistryOverride = "myprivateregistry.company.com"
         });
 
-        var redis = builder.AddRedis("redis");
+        var redis = builder.AddContainer("redis", "redis");
         builder.Build().Run();
 
         var redisManifest = await ManifestUtils.GetManifest(redis.Resource);
         var expectedManifest = $$"""
             {
               "type": "container.v0",
               "connectionString": "{redis.bindings.tcp.host}:{redis.bindings.tcp.port}",
-              "image": "myprivateregistry.company.com/{{RedisContainerImageTags.Image}}:{{RedisContainerImageTags.Tag}}",
+              "image": "myprivateregistry.company.com/redis:latest",
```

Comment:
Not sure I'm following this change. Do we build that image as part of our build? If so, what is wrong with having it be strongly typed?

Reply:
This isn't a redis test. All of the tests that aren't redis specific I am going to replace.